### PR TITLE
Add validation for form group

### PR DESCRIPTION
**Pull request by samageloff**

### CODE REVIEW

**Comment by jivinivan** [comment]
[(3) Reactions: Add validation for form group](https://trello.com/c/AM3u5Jh0/1132-3-reactions-add-validation-for-form-group)

File: test/DatePicker.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe('DatePicker', () => {
     expect(wrapper.state('value')).to.equal(firstDay)
   })
 
-  it('should run the changeCallback on changing a year', () => {
+  it.skip('should run the changeCallback on changing a year', () => {
```
Comment (samageloff) [inline test/DatePicker.js]: Temporary. Have reached out directly to @fieldsco for an assist.

Comment (jivinivan) [reply test/DatePicker.js]: @samageloff please add a card and a code comment with card URL

Comment (samageloff) [reply test/DatePicker.js]: ⚡️ 'd 🙇  @jivinivan

**Pull request merged**